src/main/kotlin/org/lunakoly/quicklink/ui/Helpers.kt

```diff
@@ -43,7 +43,7 @@ fun Project.toast(message: String) {
         .setFadeoutTime(FADE_OUT_DELAY_MS)
         .createBalloon()
         .show(
-            RelativePoint.getCenterOf(statusBar.component),
```

Comment:
I'm not sure when exactly `statusBar.component` is `null`, but maybe we should do `val component = statusBar.component ?: return` or something similar? Maybe with a regular `println()` or some other log.

I'm worried that there may be strange cases when this would crash and the user would start seeing the blinking red circle in the bottom right corner of the editor, which could be annoying. It's probably not a severe issue if some toast is not displayed in a split-second incorrect editor state.

Reply:
> I'm not sure when exactly statusBar.component is null

Not sure about that. But the code wont compile if using later versions of Intellij platform. 


> Maybe with a regular println() or some other log.

I like this idea, but where does it log to. or where does the `println` print to ?

Reply:
> I like this idea, but where does it log to. or where does the `println` print to?

It would probably make the message only visible to us in the console while we are playing with the debug version of the plugin. If we can't deliver the message to the user, then we can't deliver the message to the user :) Until they also launch IntelliJ via CLI: then they will also see it

Reply:
Pushed the changes. PTAL

Reply:
Thank you!